Playground/RxSwiftExtPlayground.playground/Pages/UIScrollView.reachedBottom.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,52 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import RxSwift
+import RxCocoa
+import RxSwiftExt
+import PlaygroundSupport
+import UIKit
+
+/*:
+ ## reachedBottom
+
+ The `reachedBottom` operator provides a sequence that emits every time scrollview is scrolled to the bottom.
+
+ Please open the Assistant Editor (⌘⌥⏎) to see the Interactive Live View example.
+ */
+
+final class ReachedBottomViewController: UITableViewController {
+    private let dataSource = Array(stride(from: 0, to: 28, by: 1))
+    private let identifier = "identifier"
+    private let disposeBag = DisposeBag()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: identifier)
+        tableView.rx.reachedBottom(offset: 40)
+            .bind { print("Reached bottom") }
```

Comment:
If you're not performing actual binding to something, I'd use `subscribe` as the intent is more explicit. 

```suggestion
            .subscribe { print("Reached bottom") }
```

Reply:
`bind` is official RxCocoa operator. The only difference with `subscribe(onNext:)` - it will crash the app with in case of an error in debug mode. I can't see any difference in code clearance here.  I would prefer to leave `bind`.

Reply:
I asked using `subscribe`, not `subscribe(onNext:)`. That's what all other examples in this repo use, please change to it.

Reply:
PS bind isn't an "operator". It's an `alias` to `bind`.

Reply:
But you are not binding to something here. `subscribe` suits better to the context.

Reply:
Lol, guys, maybe I didn't understand documentation right. Let's try to read it together
```
       /**
        Subscribes an element handler to an observable sequence.
        In case error occurs in debug mode, `fatalError` will be raised.
        In case error occurs in release mode, `error` will be logged.
    
        - parameter onNext: Action to invoke for each element in the observable sequence.
        - returns: Subscription object used to unsubscribe from the observable sequence.
        */
        public func bind(onNext: @escaping (Self.Element) -> Void) -> Disposable
```
I can see this in RxCocoa sources. 
>  Action to invoke for each element in the observable sequence.

What's the problem?

Reply:
I prefer not spending more time bike-shedding this. This is how other examples in this repo do it, please change your examples. Thanks.

Reply:
@freak4pc sorry, but i don't think it's `bike-shedding`. I came here to contribute and to make someone's life easier. I fulfill guidelines and swiftlint, added tests, docs and examples.
And here you come with secret rules, which i can get only for you. Official docs tell me: "hey man, here is function, use it". You tell me: "Sorry, it's banned". Of course, I will do like you said, but, where can i find this rule "not use `bind(onNext:)`"? Maybe you should also add it to contributing.md? Such comments about secret subjective rules make contributing very complicated.
Thanks :)

Reply:
1. You're not using `bind(onNext:)`, you're using `bind {}`.
2. `bind` operators are an alias to `subscribe(onNext:)` (read the source code). The former is meant to specifically be used for binding to UI elements, which isn't what you're doing here. It's a contextual addition, not a practical one. 
3. As I said for 2 times already - if all examples in this repo use one variation, we want to keep it aligned throughout all examples, regardless of your own personal preference. I'm not going to add every tiny thing to the contribution guides, that's why PR reviews exist. 

If you're not willing to make the modifications, we can do it.

Reply:
bind(onNext:) == bind {} Trailing closure syntax.

Reply:
Change it

Reply:
Sorry, I thought that it's bind and subscribe. Basically since there are no errors inside UI code anyways, it doesn't make a difference. Also, as I mentioned before, that's how other code in this repo does it. Thanks.